assets/constants.ts

```diff
@@ -556,6 +556,12 @@ export = {
 
   "SKILL_DIFFICULTIES": ["Easy", "Medium", "Hard"],
 
+  "SKILL_DIFFICULTY_LABEL_TO_FLOAT": {
```

Comment:
In that case, can you add a  comment on top of this mentioning that the medium difficulty value corresponds to its constant value?

Reply:
Not sure what you mean by this?

Reply:
Basically, add a comment here saying that the medium difficulty value should correspond to that in feconf. (In case we change this in the future, both should correspond)

Reply:
Ah, gotcha. Updated.